spec/requests/service_manual_topic_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.describe "Service Manual topic" do
+  describe "GET show" do
```

Comment:
This should be `GET service_manual_topic`

Reply:
Thanks, fixed.